backend/api/Database/Models/RobotModel.cs

```diff
@@ -10,4 +22,52 @@ public enum RobotModel
         AnymalX,
         AnymalD,
     }
+
+    public class RobotModel
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The type of robot model
+        /// </summary>
+        [Required]
+        //Save robot type as string in database
+        [Column(TypeName = "nvarchar(56)")]
+        public RobotType Type { get; set; }
+
+        /// <summary>
+        /// Lower battery warning threshold in percentage
+        /// </summary>
+        [Required]
+        public float BatteryWarningThreshold { get; set; }
```

Comment:
Do we want this to be required?

Reply:
I think you might have a point, it does not need to be required